docs/calicoctl/pool.md

```diff
@@ -0,0 +1,145 @@
+> manages the IP address pools and assignment.  
+
+This sections describes the `calicoctl pool` commands.
+
+These commands allow users to define and view IP address pools from which endpoint 
+IP addresses are allocated.  Users can add, remove, update, or view the IP pools.
+
+Read the [calicoctl command line interface user guide](../calicoctl.md) for a 
+full list of calicoctl commands.
+
+## Displaying the help text for 'calicoctl pool' commands
+
+Run `calicoctl pool --help` to display the following help menu for the 
+calicoctl pool commands.
+
+```
+
+Usage:
+  calicoctl pool add <CIDRS>... [--ipip] [--nat-outgoing]
+  calicoctl pool remove <CIDRS>...
+  calicoctl pool range add <START_IP> <END_IP> [--ipip] [--nat-outgoing]
+  calicoctl pool show [--ipv4 | --ipv6]
+
+Description:
+  Configure IP Pools
+
+Options:
+  --ipv4          Show IPv4 information only
+  --ipv6          Show IPv6 information only
+  --nat-outgoing  Apply NAT to outgoing traffic
+  --ipip          Use IP-over-IP encapsulation across hosts
+ 
+```
+
+## calicoctl pool commands
+
+
+### calicoctl pool add
+This command is used to add or modify subnet CIDR pools in Calico.
+
+The command can be run on any Calico node.
+
+Command syntax:
+
+```
+calicoctl pool add <CIDRS>... [--ipip] [--nat-outgoing]
+
+    <CIDRS>: A single or list of subnet cidrs separated by spaces.
+
+    --ipip: Use IP-over-IP encapsulation across hosts.
+    --nat-outgoing: Apply a NAT to outgoing traffic.
+```
+
+When a container is added to Calico networking (using Docker default networking), 
+an IP address from a pool must be allocated and assigned to the container from 
+one of the Calico pools. The allocated IP address provides a network endpoint to 
+the container.
+
+Examples:
+
+```
+# Add a pool to Calico
+$ calicoctl pool add 192.168.0.0/16
+
+# Add two pools to Calico with IP-over-IP encapsulation and NAT
+$ calicoctl pool add 192.168.0.0/16 172.24.10.0/24 --ip --nat-outgoing
+```
+
+### calicoctl pool remove
+This command is used to remove configured subnet CIDR pools from Calico.
+
+The command can be run on any Calico node.
+
+Command syntax:
+
+```
+calicoctl pool remove <CIDRS>...
+
+    <CIDRS>: A single or list of subnet cidrs separated by spaces.
+```
+
+Examples:
+
+```
+$ calicoctl pool remove 172.24.10.0/24
+```
+
+### calicoctl pool range add <START_IP> <END_IP> 
+This command adds all IP addresses between two IPs as Calico pool(s).
+
+This command can be run on any Calico node.
+
+Command syntax:
+
+```
+calicoctl pool range add <START_IP> <END_IP> [--ipip] [--nat-outgoing]
+
+    <START_IP>: IP to include from beginning of pool range.
+    <END_IP>: IP to include as the final IP in the pool range.
+    
+    --ipip: Use IP-over-IP encapsulation across hosts.
+    --nat-outgoing: Apply a NAT to outgoing traffic.
+```
+
+IP pools are added to Calico based on the IPs within the range specified.
+
+Examples:
+
+```
+$ calicoctl pool range add 172.24.0.0 172.24.25.255
```

Comment:
Add `calicoctl pool show` to this example so they see what happens when you run this command.